plugins/modules/route53_health_check.py

```diff
@@ -120,10 +142,35 @@
     weight: 100
     health_check: "{{ my_health_check.health_check.id }}"
 
+- name: create a simple health check with health_check_name as unique identifier
+  route53_health_check:
```

Comment:
Please use FQDN for example:  community.aws.route53_health_check

Reply:
@alinabuzachis what did you mean?  
this example L145-152 uses already `fqdn: ansible.com`.

Reply:
@markuman I mean fqdn for module's name. `community.aws.route53_health_check`

Reply:
Fixed. :)